catalog/spatial/csw/spatial-csw-common/pom.xml

```diff
@@ -178,5 +178,17 @@
             <groupId>ddf.security.encryption</groupId>
             <artifactId>security-encryption-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.geotools</groupId>
+            <artifactId>gt-epsg-hsql</artifactId>
+            <version>${org.geotools.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.geotools</groupId>
+            <artifactId>gt-referencing</artifactId>
+            <version>${org.geotools.version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
I think these dependencies aren't needed anymore. Looks like the changes in this PR were originally worked under the `spatial-csw-*` dirs but were eventually moved. These dependencies are the only changes under the `spatial-csw-*` dirs.

Reply:
these are still needed as test dependencies